javaagent/docker-compose.yml

```diff
@@ -8,14 +8,14 @@ services:
       # Logs are disabled by default
       OTEL_LOGS_EXPORTER: "otlp"
       # Optional specify file configuration instead of using environment variable scheme
-      # To use, call "export OTEL_CONFIG_FILE=/sdk-config.yaml" before calling docker compose up
-      OTEL_CONFIG_FILE:
+      OTEL_EXPERIMENTAL_CONFIG_FILE: /sdk-config.yaml
```

Comment:
i figured this behavior could be the default in the example, but I can return it to being optional if we want